website/content/docs/style-guide.md

```diff
@@ -0,0 +1,295 @@
+---
```

Comment:
The Kubernete's style guide doesn't apply super well, I think we'll forego this document entirely.